layouts/joomla/content/full_image.php

```diff
@@ -20,18 +20,22 @@
 	return;
 }
 
-$imgfloat = empty($images->float_fulltext) ? $params->get('float_fulltext') : $images->float_fulltext;
+$imgfloat  = empty($images->float_fulltext) ? $params->get('float_fulltext') : $images->float_fulltext;
+$extraAttr = '';
+$img       = HTMLHelper::cleanImageURL($images->image_fulltext);
 
-$img = HTMLHelper::cleanImageURL($images->image_fulltext);
-
-// Set lazyloading by default
-$img->attributes['loading'] = 'lazy';
+// Set lazyloading only for images that there's a record of width and height
+if ((isset($img->attributes['width']) && (int) $img->attributes['width'] > 0)
+&& (isset($img->attributes['width']) && (int) $img->attributes['width'] > 0))
```

Comment:
```suggestion
&& (isset($img->attributes['height']) && (int) $img->attributes['height'] > 0))
```
Width has already been checked in the previous line

Reply:
@richard67 please commit the changes directly